toolkit/resources/assets/isomaker/iso_root_arch-dependent_files/arm64/isolinux/isolinux.cfg

```diff
@@ -4,4 +4,4 @@ totaltimeout 1
 default Installer
 label Installer
 	kernel vmlinuz
-	append initrd=initrd.img root=/dev/ram0 loglevel=3
\ No newline at end of file
+	append initrd=initrd.img root=/dev/ram0 loglevel=3 console=tty0 console=ttyS0,115200n8
```

Comment:
By change, were you able to test this on ARM64 hardware?
On Azure ARM64, the serial console is different: `console=ttyAMA0 earlycon=pl011,0xeffec000`

Reply:
Thanks Chris, I did get several "failed to load Boot000..." errors on an ARM VM.  I'll try your suggestion locally and follow-up with another PR

Reply:
It turns out that for supported ARM systems (UEFI), only grub.cfg is used (not isolinux.cfg).  In this case, the x86 parameters show up for "cat /proc/cmdline", and although "systemctl status serial-getty@ttyS0.service" shows inactive (dead), "systemctl status serial-getty@ttyAMA0.service" shows active (running).  Serial console is working fine on ARM despite and regardless of my change.  Resolving this conversation.